airbyte-integrations/connectors/destination-bigquery/Dockerfile

```diff
@@ -13,7 +13,6 @@ FROM airbyte/integration-base-java:dev
 WORKDIR /airbyte
 
 ENV APPLICATION destination-bigquery
-ENV APPLICATION_VERSION 0.6.8
```

Comment:
The version information is passed in to the connector container by the worker now.